### PR TITLE
feat: antibiotics treatment episode as table

### PR DESCRIPTION
**Pull request by 9sneha-n**
### :pushpin: References

-   **Issue:** Closes #? https://app.clickup.com/t/86953e1c3, #86953e1c3

### :memo: Implementation
1. Identify Antibiotics Treatment Hospital Episode Section
2. Parse the header row from the questions by grouping 
3. Identify column questions and details questions - all questions are in column except "Please specify Other" on other select
4. Display column questions in column, and Please specify Other as row. 
5. Fix edit bug in Prevalence Mortality forms

### :video_camera: Screenshots/Screen capture

https://github.com/user-attachments/assets/8222d465-34ad-4a2d-b95b-269bc4c01e8c

### CODE REVIEW

**Comment by ifoche** [comment]
Task linked: [CU-86953e1c3 Display antibiotics in "Antibiotic treatments during hospital episode" in discharge from as a table](https://app.clickup.com/t/86953e1c3)

**Review by MiquelAdell — APPROVED** [review]
all ok by me

File: src/webapp/components/survey/TableSection.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+                </TableHead>
+
+                <TableBody>
+                    {questionGroups?.map(questionGroup => (
+                        <>
+                            <DataTableRow key={`columns-${questionGroup.groupId}`}>
+                                {questionGroup.columnQuestions.map(
+                                    question =>
+                                        question.isVisible && (
+                                            <DataTableCell key={question.code}>
```
Comment (anagperal) [inline src/webapp/components/survey/TableSection.tsx:59]: (code-only review) This part of the code is very similar to the code below, and also to the code in SurveySection.tsx. If you have time, maybe would be interesting to merge and create just one component to be use in TableSection and in SurveySection components

**Review by anagperal — CHANGES_REQUESTED** [review]
thanks @9sneha-n ! All good, I have added one minor comment that if there is no time it's not needed to be done (code-only review)

**Review by ifoche — APPROVED** [review]
thanks @9sneha-n

**Comment by ifoche** [comment]
Just a quick note, we are merging it as the procedure for generating staging versions is unclear and we have to create a new release

**Pull request merged**